improver/calculate_sleet_prob.py

```diff
@@ -31,13 +31,15 @@
 """A plugin to calculate probability of sleet"""
 
 import numpy as np
+import warnings
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "import warnings" should be placed before "import numpy as np"](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4880834)